fiware_server.js

```diff
@@ -106,27 +105,38 @@ OAuth.registerService('fiware', 2, null, function(query) {
  * @return  {Object}              The response from the token request (see above)
  */
 const getTokens = function(config, query) {
-
+  // Endpoint for requestin access token
   const endpoint = config.rootURL + '/oauth2/token';
 
+  // Sets dynamic header with clientId and Secret
+  const authHeader = toBase64(`${config.clientId}:${config.secret}`)
```

Comment:
This code is repeated in a few places. Could you extract `${config.clientId}:${config.secret}` to a function?